### ISSUE TITLE
Accessibility tests fail when run automatically via the tugboat UI

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#4025]
**Describe the defect**
Tracking issue - tests were failing due to a bug in yaml-tests. @ElijahLynn and I co-worked and tracked down the issue and found a resolution that was merged in #4023
